src/cruiseShip.js

```diff
@@ -0,0 +1,39 @@
+(function exportCruiseShip() {
```

Comment:
All component js files (and their associated test files) like this should be named in PascalCase